public/README.me

```diff
@@ -0,0 +1,7 @@
+# Front end Code
```

Comment:
What is this `README.me` file? I've never seen this file format and there is a blank `README.md`, as well